requirements/default.txt

```diff
@@ -1,4 +1,4 @@
 pytz>dev
 billiard>=3.6.0,<4.0
-kombu>=4.6.1,<5.0
```

Comment:
revert this.

Reply:
this is what is breaking tests

Reply:
well, let I am going to release kombu 4.6.2 than

Reply:
when?

Reply:
Kombu updated